src/python/elements.cpp

```diff
@@ -216,6 +216,21 @@ void init_elements(py::module& m)
         )
     ;
 
+    py::class_<SoftSolenoid, elements::Thick>(me, "SoftSolenoid")
+        .def(py::init<
+                 amrex::ParticleReal const,
+                 amrex::ParticleReal const,
+                 amrex::Vector<amrex::ParticleReal>,
+                 amrex::Vector<amrex::ParticleReal>,
+                 int const,
+                 int const
+             >(),
+             py::arg("ds"), py::arg("bscale"), py::arg("cos_coefficients"), py::arg("sin_coefficients"),
```

Comment:
Better readable long line:
```suggestion
             py::arg("ds"), py::arg("bscale"),
             py::arg("cos_coefficients"), py::arg("sin_coefficients"),
```